Sources/PointFreeRouter/Routes.swift

```diff
@@ -389,6 +396,7 @@ private let subscriberDataIso = PartialIso<String, SubscribeData?>(
     if let referralCode = data.referralCode?.rawValue {
       parts.append("\(SubscribeData.CodingKeys.referralCode.rawValue)=\(referralCode)")
     }
+    parts.append("\(SubscribeData.CodingKeys.useRegionalDiscount.rawValue)=\(data.useRegionalDiscount)")
```

Comment:
Should we only append this if it's true?

Reply:
yep can do...

Reply:
After reviewing all the snaps it should reduce some churn there, too.